test-runtime/resolve.test.ts

```diff
@@ -15,37 +14,45 @@ import {
   unitNameRegex
 } from './util';
 
+declare const page: Page;
+
 for (const type of selectionTypes) {
-  const embed = embedFn(browser);
+  const embed = embedFn(page);
   const isInterval = type === 'interval';
   const hits = isInterval ? hitsMaster.interval : hitsMaster.discrete;
   const fn = isInterval ? brush : pt;
 
-  describe(`${type} selections at runtime`, () => {
+  describe(`${type} selections at runtime`, async () => {
+    beforeAll(async () => {
+      await page.goto('http://0.0.0.0:8000/test-runtime/');
+    });
+
     compositeTypes.forEach(specType => {
-      const testRender = testRenderFn(browser, `${type}/${specType}`);
-      describe(`in ${specType} views`, () => {
-        // Loop through the views, click to add a selection instance.
-        // Store size should stay constant, but unit names should vary.
-        it('should have one global selection instance', () => {
+      const testRender = testRenderFn(page, `${type}/${specType}`);
+      describe(`in ${specType} views`, async () => {
+        /**
+         * Loop through the views, click to add a selection instance.
+         * Store size should stay constant, but unit names should vary.
+         */
+        it('should have one global selection instance', async () => {
           const selection = {
             type,
             resolve: 'global',
             ...(specType === 'facet' ? {encodings: ['y']} : {})
           };
 
           for (let i = 0; i < hits[specType].length; i++) {
-            embed(spec(specType, i, selection));
+            await await embed(spec(specType, i, selection));
```

Comment:
`await await`?

Reply:
Ups

Reply:
Fixed in https://github.com/vega/vega-lite/pull/4506